app/controllers/players_controller.rb

```diff
@@ -145,7 +145,7 @@ def tracking
     @players = Player.limit(@show_limit.to_i).where("overall_ehp > 250 OR player_name IN #{Player.sql_supporters}").where(player_acc_type: @filters.keys).where("overall_ehp_day_start > 0").order(ordering)
 
     if @restrictions["10 hitpoints"]
-      @players = @players.where(hitpoints_lvl: 10)
+      @players = @players.where(hitpoints_lvl: 10).where("combat_lvl > 3")
```

Comment:
I think `combat_level` is a float in the database, and fresh accounts are at 3.4, so we may need to update this

Reply:
My bad, I did some testing but must have missed that I guess. I'll update it when I can :)